stablehlo/reference/Ops.cpp

```diff
@@ -30,13 +33,106 @@ namespace mlir {
 namespace stablehlo {
 namespace {
 
+static Type getF64Type(MLIRContext *context) {
+  return mlir::Builder(context).getF64Type();
+}
+
+static Type getI64Type(MLIRContext *context) {
+  return mlir::Builder(context).getI64Type();
+}
```

Comment:
Given that these functions are used only once, let's inline them into their callsites?